src/transformers/modeling_utils.py

```diff
@@ -2306,7 +2306,7 @@ def save_pretrained(
             files_timestamps = self._get_files_timestamps(save_directory)
 
         # Only save the model itself if we are using distributed training
-        model_to_save = unwrap_model(self)
+        model_to_save = unwrap_model(self) if is_accelerate_available() else Accelerator().unwrap_model(self)
```

Comment:
Not sure what is the best case here. I don't think we want to force users to install accelerate to save a model. If they are saving after training through trainer or accelerate, they will have accelerate installed.